bucket_manager.go

```diff
@@ -127,3 +130,7 @@ func (bm *BucketManager) GetBucketID(ds Ds, name BucketName) (BucketId, error) {
 		return bucket.Id, nil
 	}
 }
+
+func (bm *BucketManager) close() error {
+	return bm.fd.Close()
```

Comment:
`bucket.Meta` can't be removed when we call `os.RemoveAll` on windows because it not closed.